pkg/storage/export/export/export_test.go

```diff
@@ -1400,6 +1400,7 @@ var _ = Describe("Export controller", func() {
 		Expect(dvs).To(HaveLen(1))
 		Expect(dvs[0]).ToNot(BeNil())
 		Expect(dvs[0].Name).To((Equal("pvc")))
+		Expect(dvs[0].Spec.PVC).To(BeNil())
```

Comment:
this should probably be in previous commit

Reply:
yup moved